### ISSUE TITLE
Add tfx example with custom estimator

### ISSUE THREAD

**Issue by DurivetMatthias** [tensorflow/tfx#127]
I'm having trouble implementing tfx with a custom estimator.
The pipeline runs start-to-end and creates a new model on ML Engine (AI platform).

However when I try to make a prediction, it returns {'predictions': []}
[link](https://github.com/tensorflow/tfx/issues/114#issue-443364979) to that issue.

I imagine almost everyone trying to seriously implement tfx in a buisness context will use a custom estimator over a canned estimator.

In short, it seems useful to add an example that uses a custom estimator.

**Comment by DurivetMatthias** [comment]
I have resolved the underlying issue, turns out it wasn't the fault of the custom estimator. That being said I still think its a good idea to have atleast one example with a custom estimator

**Comment by ruoyu90** [comment]
@DurivetMatthias Thanks for the feedback and glad the issue has been resolved. By custom estimator example, do you mean an example to create custom estimator or an example to use custom estimator? If it's the former one, you might find [this](https://www.tensorflow.org/guide/custom_estimators) link useful. To use a custom estimator in the trainer, all you want to do is to follow the previous link and put the logic into the module file similar to [_build_estimator()](https://github.com/tensorflow/tfx/blob/master/tfx/examples/chicago_taxi_pipeline/taxi_utils.py#L148) here.

**Comment by DurivetMatthias** [comment]
As someone new to both tfx and custom estimators it was hard for me to distinguish between what code is for the former or the latter. Now I've been trough my project I can see they are completely separate. What happens with this issue is up to you. (I originally meant an example of tfx that uses a custom estimator)

**Comment by rmothukuru** [comment]
Closing the issue as it has been resolved.
